UnityProject/Assets/Scripts/Objects/Closets/ClosetControl.cs

```diff
@@ -5,14 +5,18 @@
 using Mirror;
 using AddressableReferences;
 using Items;
+using MiniGames;
+using NaughtyAttributes;
 using Objects.Atmospherics;
+using UnityEngine.Events;
+using Random = UnityEngine.Random;
 
 namespace Objects
 {
 	/// <summary>
 	/// Allows closet to be opened / closed / locked
 	/// </summary>
-	public class ClosetControl : NetworkBehaviour, IServerSpawn, ICheckedInteractable<PositionalHandApply>, IRightClickable, IExaminable, IEscapable
+	public class ClosetControl : NetworkBehaviour, IServerSpawn, ICheckedInteractable<PositionalHandApply>, IRightClickable, IExaminable, IEscapable, IMiniGame
```

Comment:
I would rather make a new component with the minigame that uses ``ClosetControl``. This minigame also won't be used in most of crates/lockers, so it is better to keep the responsibility of this class scoped. The abandoned crates would be their own prefab which add this component and the minigame component.

TLDR: invert the references so a component that implements ``IMinigame`` uses ``ClosetControl`` and not the other way.

Reply:
On it when I jump on rider 👍